app/validation.py

```diff
@@ -60,11 +61,12 @@ def get_validator(schema_name, enforce_required=True, required_fields=None):
     if enforce_required:
         schema = _SCHEMAS[schema_name]
     else:
-        schema = _SCHEMAS[schema_name].copy()
+        schema = copy.deepcopy(_SCHEMAS[schema_name])
```

Comment:
Is there any reason to replace it other than that we can? 
It still might cause the same kind of problem in the future.